bodhi/server/models.py

```diff
@@ -554,6 +555,54 @@ def get_pkg_pushers(self, branch, settings):
 
         return (committers, watchers), (committergroups, watchergroups)
 
+    def get_pkg_committers_from_pagure(self, settings):
```

Comment:
You should be able to just use ```bodhi.server.config.config``` instead of passing the settings to this method. The config is already imported at the top of this module.